data.sql

```diff
@@ -13,4 +13,32 @@ INSERT 0 1
 vet_clinic=# INSERT INTO animals (ID,Name,Date_of_birth,Weight_kg,Neutered,Escape_attempts)
 vet_clinic-# VALUES (4, 'Devimon', 'May/12/2017', 11, True, 5);
 INSERT 0 1
-vet_clinic=# 
\ No newline at end of file
+vet_clinic=# 
+
+vet_clinic=# INSERT INTO animals (ID,Name,Date_of_birth,Weight_kg,Neutered,Escape_attempts)
+VALUES (5, 'Chaemander', 'Feb/8/2020', -11, False, 0);
+INSERT 0 1
+
+vet_clinic=# INSERT INTO animals (ID,Name,Date_of_birth,Weight_kg,Neutered,Escape_attempts)
+VALUES (6, 'Plantmon', 'Nov/15/2021', -5, True, 2);
+INSERT 0 1
+
+vet_clinic=# INSERT INTO animals (ID,Name,Date_of_birth,Weight_kg,Neutered,Escape_attempts)
+VALUES (7, 'Squirtle', 'Apr/2/1993', -12.13, False, 3);
+INSERT 0 1
+
+vet_clinic=# INSERT INTO animals (ID,Name,Date_of_birth,Weight_kg,Neutered,Escape_attempts)
+VALUES (8, 'Angemon', 'Jun/12/2005', -45, True, 1);
+INSERT 0 1
+
+vet_clinic=# INSERT INTO animals (ID,Name,Date_of_birth,Weight_kg,Neutered,Escape_attempts)
+VALUES (9, 'Boarmon', 'Jun/7/2005', 20, True, 7);
+INSERT 0 1
+
```

Comment:
Great job @zakiyousufi 🤝

- [ ] Kindly make all your `weight_kg` values floating numbers as they are provided in the project requirements.

Reply:
This change has been done well